src/tools/status_tool.cpp

```diff
@@ -727,23 +728,42 @@ int StatusTool::PrintChunkserverStatus(bool checkLeftSize) {
             }
             uint64_t size = chunkNum * FLAGS_chunkSize;
             chunkLeftSize.emplace_back(size / mds::kGB);
-            // walfilepool left size
-            metricName = GetCSLeftWalSegmentName(csAddr);
-            uint64_t walSegmentNum;
-            res = metricClient_->GetMetricUint(csAddr, metricName,
-                                                    &walSegmentNum);
+
+            // check whether use chunkfilepool as walpool from chunkserver conf metric  // NOLINT
+            std::string metricValue;
+            metricName = GetUseChunkFilePoolAsWalPoolName(csAddr);
+            res = metricClient_->GetConfValueFromMetric(csAddr, metricName,
+                                                        &metricValue);
             if (res != MetricRet::kOK) {
-                std::cout << "Get left wal segment size of chunkserver "
-                          << csAddr << " fail!" << std::endl;
+                std::cout << "Get use chunkfilepool as walpool conf "
+                        << csAddr << " fail!" << std::endl;
                 ret = -1;
                 continue;
             }
-            size = walSegmentNum * FLAGS_walSegmentSize;
-            walSegmentLeftSize.emplace_back(size / mds::kGB);
+            useChunkFilePoolAsWalPool = StringToBool(metricValue, useChunkFilePoolAsWalPool);
+            // walfilepool left size
+            if (!useChunkFilePoolAsWalPool) {
```

Comment:
There are two situations where walpool is not used, only useChunkFilePoolAsWalPool is considered here

Reply:
> There are two situations where walpool is not used, only useChunkFilePoolAsWalPool is considered here

ok, I just knew. 
fixed.